### PR TITLE
registry: Only show top of dashboard when an admin user

### PR DESCRIPTION
**Pull request by stefwalter**

### CODE REVIEW

**Comment by stefwalter** [comment]
Needs tests

File: pkg/registry/scripts/main.js [hunk 1]
```diff
@@ -277,10 +283,20 @@
...
+                loadNamespace($route.current);
 
             return {
-                namespaces: calcAvailable,
+                globals: function(value) {
+                    if (arguments.length === 0)
+                        return globals;
+                    value = !!value;
+                    if (globals !== value) {
+                        console.log("changing globals", globals, value);
```
Comment (petervo) [inline pkg/registry/scripts/main.js]: Still needed?

Comment (stefwalter) [inline pkg/registry/scripts/main.js]: Nope.

**Comment by dperpeet** [comment]
looks good, but holding off on merge because @petervo is currently touching this code as well

**Pull request closed without merging**